apps/studio/components/layouts/ProjectLayout/LayoutHeader/NotificationsPopoverV2/NotificationsPopover.tsx

```diff
@@ -0,0 +1,265 @@
+                          onSelect={() => onSelectFilter(filter.id)}
+                          onClick={() => onSelectFilter(filter.id)}
+                        >
+                          <p>{filter.label}</p>
+                          {selectedFilter === filter.id && <IconCheck />}
+                        </CommandItem_Shadcn_>
+                      ))}
+                    </CommandGroup_Shadcn_>
+                  </CommandList_Shadcn_>
+                </Command_Shadcn_>
+              </PopoverContent_Shadcn_>
+            </Popover_Shadcn_>
+          </div>
+        </div>
+        <div className="border-t">
+          {isLoading && (
+            <div className="p-4">
+              <GenericSkeletonLoader />
+            </div>
+          )}
+          {isError && (
+            <div className="p-4">
+              <AlertError subject="Failed to retrieve notifications" error={error} />
+            </div>
+          )}
+          {isSuccess && (
+            <div className="flex flex-1 h-[400px]">
+              {notifications.length > 0 &&
+              !(activeTab === 'archived' && selectedFilter === 'unread') ? (
+                <InfiniteList
+                  items={notifications}
+                  ItemComponent={NotificationRow}
+                  LoaderComponent={
+                    <div className="p-4">
+                      <ShimmeringLoader />
+                    </div>
+                  }
+                  itemProps={{
+                    setRowHeight: (idx: number, height: number) => {
+                      if (rowHeights.current) {
+                        rowHeights.current = { ...rowHeights.current, [idx]: height }
+                      }
+                    },
+                    getProject: (ref: string) => projects?.find((project) => project.ref === ref),
+                    getOrganization: (id: number) => organizations?.find((org) => org.id === id),
+                    onArchiveNotification: (id: string) =>
+                      updateNotifications({ ids: [id], status: 'archived' }),
+                  }}
+                  getItemSize={(idx: number) => rowHeights?.current?.[idx] ?? 56}
+                  hasNextPage={hasNextPage}
+                  isLoadingNextPage={isFetchingNextPage}
+                  onLoadNextPage={() => fetchNextPage()}
+                />
+              ) : (
+                <div className="flex flex-col gap-y-4 items-center flex-grow justify-center">
+                  <IconInbox size={32} className="text-foreground-light" />
+                  <div className="flex flex-col gap-y-1">
+                    <p className="text-foreground-light text-sm w-64 text-center">
+                      {activeTab === 'archived'
+                        ? `No archived ${
+                            selectedFilter !== undefined ? `${selectedFilter} ` : ''
+                          }notifications`
```

Comment:
Copy is a bit weird when "All Notifications" filter is on and it becomes "No archived all notifications". Maybe just "No archived notifications" would be better.
<img width="482" alt="Screenshot 2024-01-04 16 21 01@2x" src="https://github.com/supabase/supabase/assets/1732217/864fa53d-74e5-42af-bbf2-243ddc569a95">

Reply:
oh whoops good catch this!